lib/elixir/lib/module/checker.ex

```diff
@@ -1,16 +1,212 @@
 defmodule Module.Checker do
-  def verify(module_map, _binary) do
+  @moduledoc false
+
+  defmodule Coordinator do
+    @moduledoc false
+
+    def start_link(maps) do
+      ets = :ets.new(:checker_cache, [:set, :public])
```

Comment:
We should always pick one of read_concurrency or write_concurrency,

Reply:
I guess we will read more than we will write, so `read_concurrency`?

Reply:
I think so. All writes are also serial so far I think?

Reply:
Yeah